### DISCUSSION TITLE
replace `envd destroy` with `envd down`

### DISCUSSION THREAD

**Discussion by Triple-Z** [tensorchord/envd#426] [Ideas]
ref: https://github.com/tensorchord/envd/pull/422#issuecomment-1159026216

> maybe down is a better word than destroy:
> 
> 1. docker-compose uses it.
> 2. down is the antonym for up.
> 3. destroy is easy to typo :(

**Comment by gaocegege** [comment]
We will stop and remove the container if we run `envd destroy`, if we use down, I am afraid that users may think the container is just shutdown, and not be removed.

cc @terrytangyuan @VoVAllen

**Reply by Triple-Z** [reply]
In fact, [`docker-compose down`](https://docs.docker.com/compose/reference/down/) also removes the containers.

**Reply by gaocegege** [reply]
LGTM

**Reply by terrytangyuan** [reply]
Sounds good.

**Comment by gaocegege** [comment]
I think we should merge it in 0.2.0 since it has the breaking change.

**Reply by Triple-Z** [reply]
Agreed.

**Reply by popfido** [reply]
Or support both `destroy` and `down` to keep backward compatibility.

**Comment by VoVAllen** [comment]
To me, `down` is more likely `halt` instead of `destroy`. 

I think `down` is an ambiguous term. I prefer `stop`/`halt` + `destroy` over `down`

Reference:
- AWS: `stop` vs `terminate`
- vagrant: `halt` vs `destroy`
- docker-compose: `stop` vs `down`

**Reply by Triple-Z** [reply]
cc @gaocegege

**Reply by gaocegege** [reply]
I think it is great to keep two cmds. One is for shutdown the ctr. The other is to remove them.

I have no objection to the name

**Comment by gaocegege** [comment]
How about stop and destroy?

**Reply by Triple-Z** [reply]
Change `pause` to `stop`?

**Reply by gaocegege** [reply]
Pause/resume actually freezes/resumes the processes in the container. Stop kills the processes. It is slightly different. :thinking:

**Comment by Triple-Z** [comment]
As the discussions above, can we add the `envd down` command as the alias of `envd destroy` in the future versions?

**Reply by gaocegege** [reply]
SGTM!
